docs/sphinx/dev_guide/contributing.rst

```diff
@@ -153,24 +174,26 @@ be run on PRs that are made on branches within the GitHub RAJA repository.
 
           If successful, this will create a branch in your local copy of the
           RAJA repo labeled ``pr-from-fork/<PR #> and you will be on that
-          local branch. To verify this, you can run::
+          local branch in your checkout space. To verify this, you can run
+          the following command after you run the script::
 
             $ git branch
 
           You will see the new branch in the listing of branches and the branch
-          you are on will be starred.  
+          you are on will be starred.
 
-          You can push the new branch to the RAJA repo on GitHub by running::
+          You can push the new branch to the RAJA repo on GitHub::
 
             $ git push origin <branch-name>
 
-          and make a PR for the new branch. It is good practice to reference the
-          original PR in the description of this new PR to track the original
-          PR discussion and reviews.
+          and make a PR for the new branch. It is good practice to reference 
+          the original PR in the description of the new PR to track the 
+          original PR discussion and reviews.
+
+          All CI checks will be triggered to run on the PR make in the
+          RAJA repo. When everything passes and the PR is approved, it may 
```

Comment:
The first sentence has a typo, but I'm not sure how it's supposed to be structured

Reply:
Fixed.